node_modules/oae-principals/lib/api.user.js

```diff
@@ -774,3 +809,225 @@ var _setAdmin = function(ctx, adminType, isAdmin, principalId, callback) {
         return PrincipalsDAO.setAdmin(adminType, isAdmin, principalId, callback);
     });
 };
+
+/**
+ * Send an email token to a user that can be used to verify the user owns the email address
+ *
+ * @param  {Context}    ctx             Standard context object containing the current user and the current tenant
+ * @param  {User}       user            The user to send the email token to
+ * @param  {String}     email           The email address where to send the token to
+ * @param  {String}     [token]         The token to send. If left null, a new one will be generated
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Object}     callback.err    An error that occurred, if any
+ * @api private
+ */
+var _sendEmailToken = function(ctx, user, email, token, callback) {
+    callback = callback || function(err) {
+        if (err) {
+            log().error({'err': err, 'userId': user.id}, 'Unable to send a user a verification email');
+        }
+    };
+
+    // Generate a token if none was specified
+    if (!token) {
+        token = ShortId.generate();
+    }
+
+    // Store the token
+    PrincipalsDAO.storeEmailToken(user.id, email, token, function(err) {
+        if (err) {
+            return callback(err);
+        }
+
+        // The EmailAPI expects a user to have a verified email address. As this is not the case
+        // when sending an email token, we send in a patched user object
+        var userToEmail = _.extend({}, user, {'email': email});
+
+        var fullTenant = TenantsAPI.getTenant(user.tenant.alias);
```

Comment:
We can just call the variable `tenant` instead of `fullTenant`, as we should strive to call things like just the tenant alias `tenantAlias`, etc...